server/src/main/java/io/crate/protocols/ConnectionStats.java

```diff
@@ -21,38 +21,101 @@
 
 package io.crate.protocols;
 
+import java.beans.ConstructorProperties;
+import java.io.IOException;
+
+import org.elasticsearch.Version;
 import org.elasticsearch.common.io.stream.StreamInput;
 import org.elasticsearch.common.io.stream.StreamOutput;
 import org.elasticsearch.common.io.stream.Writeable;
 
-import java.io.IOException;
-
 public final class ConnectionStats implements Writeable {
 
+    public static ConnectionStats EMPTY = new ConnectionStats(0, 0, 0, 0, 0, 0);
+
     private final long open;
     private final long total;
 
-    public ConnectionStats(long open, long total) {
+    private final long receivedCount;
+
+    private final long receivedSize;
+
+    private final long transmittedCount;
+
+    private final long transmittedSize;
+
+    /**
+     *
+     */
```

Comment:
If we keep composite object in the mbean, I want to add a comment here, since this class is not in the jmx module but is also used (ctor annotation) as a composite object for jmx exposed attributes.